app/os_macos.go

```diff
@@ -557,6 +560,19 @@ func gio_onMouse(view, evt C.CFTypeRef, cdir C.int, cbtn C.NSInteger, x, y, dx,
 	})
 }
 
+//export gio_onExternalDrop
+func gio_onExternalDrop(view C.CFTypeRef, path *C.char) {
+	fileUrl := C.GoString(path)
+	w := mustView(view)
+
+	w.w.Event(transfer.DataEvent{
+		Type: "external-file",
```

Comment:
This should be a MIME type, as documented in the field. Doesn't the macOS API give us a type?